### ISSUE TITLE
Unhandled Exception thrown when no arguments are passed, should show help instead

### ISSUE THREAD

**Issue by samarunraj** [SteveGilham/altcover#17]
Version: altcover.3.0.466.nupkg

```dotnet C:\temp\altcover\tools\netcoreapp2.0\AltCover.dll

Unhandled Exception: System.ArgumentException: The input sequence was empty.
Parameter name: source
   at Microsoft.FSharp.Collections.SeqModule.Head[T](IEnumerable`1 source)
   at AltCover.Main.Main(String[] arguments) in C:\projects\altcover\AltCover\AltCover.fs:line 352```

BTW: Love the tool

**Comment by SteveGilham** [comment]
Fix will be in this week's release.  

The behaviour on empty is intended to be to take the defaults for `-i`, `-o` and `-x`

**Comment by SteveGilham** [comment]
Addressed in release 3.0.475
